docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -0,0 +1,1911 @@
+    }
+    ```
+
+### net_listening
+
+Indicates whether the client is actively listening for network connections.
+
+*Parameters*
+
+None
+
+*Returns*
+
+`result` (*BOOLEAN*) - `true` if the client is actively listening for network connections; otherwise `false`.
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"net_listening","params":[],"id":53}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"net_listening","params":[],"id":53}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 53,
+      "result" : true
+    }
+    ```
+
+### net_peerCount
+
+Returns the number of peers currently connected to the client.
+
+*Parameters*
+
+None
+
+*Returns*
+
+`result` : *integer* - Number of connected peers in hexadecimal.
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"net_peerCount","params":[],"id":53}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"net_peerCount","params":[],"id":53}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 53,
+      "result" : "0x5"
+    }
+    ```
+
+## Eth Methods
+
+### eth_syncing
+
+Returns an object with data about the synchronization status, or `false` if not synchronizing.
+
+*Parameters*
+
+None
+
+*Returns*
+
+`result` : *Object|Boolean* - Object with synchronization status data or `false`, when not synchronizing:
+
+* `startingBlock` : *quantity* - Index of the highest block on the blockchain when the network synchronization starts.
+
+  If you start with an empty blockchain, the starting block is the beginning of the blockchain (`startingBlock` = 0).
+
+  If you import a block file using `pantheon import <block-file>`, the synchronization starts at the head of the blockchain, and the starting block is the next block synchronized. For example, if you imported 1000 blocks, the import would include blocks 0 to 999, so in that case `startingBlock` = 1000.
+
+* `currentBlock` : *quantity* - Index of the latest block (also known as the best block) for the current node. This is the same index that [eth_blockNumber](#eth_blocknumber) returns.
+
+* `highestBlock`: *quantity* - Index of the highest known block in the peer network (that is, the highest block so far discovered among peer nodes). This is the same value as `currentBlock` if the current node has no peers.
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":51}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
```

Comment:
I don't think we need these two lines showing the wscat connection - the example request should really be the part they can copy and paste

Reply:
ok